tests/testsuite.at

```diff
@@ -105,6 +107,7 @@ check_pedit([Test generation of TrueType instructions],[test127.pe],[Ambrosia.sf
 check_pedit([Test StrSplit overrun],[test128.pe])
 check_pedit([Ligatures and Fractions Table Lookups],[test129.pe])
 check_pedit([TTC loading],[test130.pe],[NotoSans-Regular.ttc])
+check_pedit([file:// protocol],[test131.pe],[${abs_srcdir}/fonts/Ambrosia.sfd])
```

Comment:
Why doesn't passing `[Ambrosia.sfd]`  work?

Reply:
`file://` URLs are supposed to be always absolute, see <https://tools.ietf.org/html/rfc8089#page-4>. File managers will also always pass URLs with absolute paths.

Given that "support for `file://`" means just stripping the path, in practice a simple relative path would work, but just by accident. :)

Reply:
But the whole `check_file` macro ensures that it always gets transformed to an absolute path

Reply:
@jtanx: you are absolutely right. Somehow at a certain point I needed that modification (forgot why :confused:), but, indeed, it is not needed now. I'll edit the commit after the review.